Src/BootCamp.Chapter/FileCleaner.cs

```diff
@@ -0,0 +1,251 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class FileCleaner
+    {
+        private const char DollarSign = '£';
+
+        /// <summary>
+        /// Cleans up dirtyFileName 
+        /// </summary>
+        /// <param name="dirtyFile">Dirty file with "_" placed in random places.</param>
+        /// <param name="cleanedFile">Cleaned up file without any "_".</param>
+        public static void Clean(string dirtyFile, string cleanedFile)
+        {
+            if (!IsValidFile(dirtyFile, cleanedFile))
+            {
+                return;
+            }
+
+            var corruptedBalanceLines = File.ReadAllLines(dirtyFile);
+            var fixedBalanceLines = new string[corruptedBalanceLines.Length];
+
+            File.WriteAllText(cleanedFile, FixCorruptedBalanceFile(corruptedBalanceLines, fixedBalanceLines));
+
+            DisplayStasticalData(CreateBalanceLinesForTextTable(fixedBalanceLines));
+        }
+
+        private static bool IsValidFile(string dirtyFile, string cleanedFile)
+        {
+            if (dirtyFile == null || dirtyFile.Trim() == "")
+            {
+                Console.WriteLine("The corrupted input file is not found. The file path cannot be null or empty.");
+                return false;
+
+            }
+            else if (!File.Exists(dirtyFile))
+            {
+                Console.WriteLine("The corrupted input file is not found. Please check the file path.");
+                return false;
+            }
+
+            if (cleanedFile == null || cleanedFile.Trim() == "")
+            {
+                Console.WriteLine("The fixed output file is not saved. The file path cannot be null or empty.");
+                return false;
+            }
+            else if ((cleanedFile.Contains(@"\")))
+            {
+                var pathLength = cleanedFile.LastIndexOf(@"\") + 1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5116146)